### ISSUE TITLE
Could not pass BlockBreakEvent to TreeFeller v1.10.1

### ISSUE THREAD

**Issue by wilbowma** [ThizThizzyDizzy/tree-feller#6]
Using Paper 31 for Minecraft 1.15.1, and TreeFeller v1.10.1, I get the following error in the server console when I try to cut down a tree. Minecraft functions as if TreeFeller is not installed.

```
[18:26:40 ERROR]: Could not pass event BlockBreakEvent to TreeFeller v1.10.1
java.lang.NoClassDefFoundError: Could not initialize class com.thizthizzydizzy.treefeller.TreeFellerCompat
	at com.thizthizzydizzy.treefeller.TreeFeller.fellTree(TreeFeller.java:430) ~[?:?]
	at com.thizthizzydizzy.treefeller.TreeFeller.fellTree(TreeFeller.java:67) ~[?:?]
	at com.thizthizzydizzy.treefeller.TreeFeller.fellTree(TreeFeller.java:64) ~[?:?]
	at com.thizthizzydizzy.treefeller.TreeFeller.fellTree(TreeFeller.java:61) ~[?:?]
	at com.thizthizzydizzy.treefeller.TreeFeller.fellTree(TreeFeller.java:58) ~[?:?]
	at com.thizthizzydizzy.treefeller.BlockBreak.onBlockBreak(BlockBreak.java:46) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor103.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.15.1.jar:git-Paper-31]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:76) ~[patched_1.15.1.jar:git-Paper-31]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.15.1.jar:git-Paper-31]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:545) ~[patched_1.15.1.jar:git-Paper-31]
	at net.minecraft.server.v1_15_R1.PlayerInteractManager.breakBlock(PlayerInteractManager.java:313) ~[patched_1.15.1.jar:git-Paper-31]
	at net.minecraft.server.v1_15_R1.PlayerInteractManager.a(PlayerInteractManager.java:272) ~[patched_1.15.1.jar:git-Paper-31]
	at net.minecraft.server.v1_15_R1.PlayerInteractManager.a(PlayerInteractManager.java:240) ~[patched_1.15.1.jar:git-Paper-31]
	at net.minecraft.server.v1_15_R1.PlayerConnection.a(PlayerConnection.java:1319) ~[patched_1.15.1.jar:git-Paper-31]
	at net.minecraft.server.v1_15_R1.PacketPlayInBlockDig.a(SourceFile:40) ~[patched_1.15.1.jar:git-Paper-31]
	at net.minecraft.server.v1_15_R1.PacketPlayInBlockDig.a(SourceFile:10) ~[patched_1.15.1.jar:git-Paper-31]
	at net.minecraft.server.v1_15_R1.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:23) ~[patched_1.15.1.jar:git-Paper-31]
	at net.minecraft.server.v1_15_R1.TickTask.run(SourceFile:18) ~[patched_1.15.1.jar:git-Paper-31]
	at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[patched_1.15.1.jar:git-Paper-31]
	at net.minecraft.server.v1_15_R1.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.15.1.jar:git-Paper-31]
	at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[patched_1.15.1.jar:git-Paper-31]
	at net.minecraft.server.v1_15_R1.MinecraftServer.aZ(MinecraftServer.java:1037) ~[patched_1.15.1.jar:git-Paper-31]
	at net.minecraft.server.v1_15_R1.MinecraftServer.executeNext(MinecraftServer.java:1030) ~[patched_1.15.1.jar:git-Paper-31]
	at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119) ~[patched_1.15.1.jar:git-Paper-31]
	at net.minecraft.server.v1_15_R1.MinecraftServer.sleepForTick(MinecraftServer.java:1014) ~[patched_1.15.1.jar:git-Paper-31]
	at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:937) ~[patched_1.15.1.jar:git-Paper-31]
	at java.lang.Thread.run(Unknown Source) [?:?]
```

**Comment by ThizThizzyDizzy** [comment]
Thanks for the issue report;
I'll release a fix in a few minutes

**Comment by ThizThizzyDizzy** [comment]
Scratch that- to reproduce

**Comment by wilbowma** [comment]
Oh another bit of info: Using Java 11 (adoptopenjdk11-jre)

**Comment by ThizThizzyDizzy** [comment]
Fixed in 1.10.2

**Comment by wilbowma** [comment]
That works thanks!
